packages/wp-now/public/with-node-version.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+MIN_MAJOR=18
+MIN_MINOR=0
+MIN_PATCH=0
+
+VERSION=( $(node -e "console.log(/^v(\d+)\.(\d+)\.(\d+)$/.exec(process.version).slice(1, 4).join(' '))") )
+
+MAJOR=${VERSION[0]}
+MINOR=${VERSION[1]}
+PATCH=${VERSION[2]}
+
+if [ $MAJOR -lt $MIN_MAJOR ]; then
+	printf "This script is requires node version v$MIN_MAJOR.$MIN_MINOR.$MIN_PATCH or above; found v$MAJOR.$MINOR.$PATCH."
```

Comment:
```suggestion
	printf "This script is requires node version v$MIN_MAJOR.$MIN_MINOR.$MIN_PATCH or above; found v$MAJOR.$MINOR.$PATCH"
```

I find the periods to add a little of confusion to the output.

Reply:
dots removed in 374b1370